target/linux/realtek/dts-5.15/rtl9302_zyxel_xgs1250-12.dts

```diff
@@ -71,6 +72,107 @@
 							  // (5G, 10/100) (10G, 5G, 2.5G)
 		led_set2 = <0x0a20 0x0a01>; // LED set 2: 1000MBit, 10GBit
 	};
+
+	thermal-zones {
+		phy24-thermal {
+			/* Poll every 10 seconds */
+			polling-delay-passive = <10000>; /* milliseconds */
```

Comment:
You don't need to add the units, these are mentioned in the binding.

Reply:
Yeah, true. I'll keep the explanatory comments and remove the ones that are just units.